client/src/pages/Search/AddressSearch.js

```diff
@@ -22,15 +26,25 @@ export default function AddressSearch({ updateFields }) {
       componentRestrictions: { country: "usa" },
     },
   });
-  
+
+  function handleUserAddressError() {
+    dispatch(
+      alertSnackbar({
+        message: "Please Enable Location Services",
+        severity: "error",
+      })
+    );
+  }
+
   async function updateToUserAddress({ updateFields }) {
     try {
       const coordinates = await getUserCoordinates();
       const latitude = coordinates.coordinates.latitude;
       const longitude = coordinates.coordinates.longitude;
       updateFields({ longitude, latitude });
     } catch {
-      alert("Please Enable Location Services");
+      handleUserAddressError();
+      return null;
```

Comment:
no need to return anything here, since this is a void function (I see that it's being used on click)